cmd/Konfigurator/main.go

```diff
@@ -36,14 +36,19 @@ func main() {
 }
 
 func watchKonfiguratorTemplate() {
-	namespace, err := k8sutil.GetWatchNamespace()
-	if err != nil {
-		logrus.Fatalf("Failed to get watch namespace: %v", err)
-	}
+	namespace := getWatchNamespace()
 
 	watch("konfigurator.stakater.com/v1alpha1", "KonfiguratorTemplate", namespace, 15)
 }
 
+func getWatchNamespace() string {
```

Comment:
I think its better to reuse "GetWatchNamespace" method from k8sutil package. We can just make the error a warning, saying that no WATCH_NAMESPACE detected so watching in all namespaces. And the rest will work as expected

Reply:
https://github.com/operator-framework/operator-sdk/blob/121664e0093774c6d4fa4528fe7367d0848e6d74/pkg/k8sutil/k8sutil.go#L36

They log an error in their function as well which is not good as if our app logs an error but we are supporting that as a feature.